src/engraving/libmscore/score.cpp

```diff
@@ -850,9 +850,8 @@ void Score::spell()
             for (int track = strack; track < etrack; ++track) {
                 Element* e = s->element(track);
                 if (e && e->type() == ElementType::CHORD) {
-                    notes.insert(notes.end(),
-                                 toChord(e)->notes().begin(),
-                                 toChord(e)->notes().end());
+                    std::copy_if(toChord(e)->notes().begin(), toChord(e)->notes().end(),
+                                 std::back_inserter(notes), [this](Element* ce) { return selection().isNone() || ce->selected(); });
```

Comment:
This is the part that ensures "Respell pitches" only applies to selected notes, right?

Reply:
And so fixes https://musescore.org/en/node/284064?

Reply:
Apparently it is!

Reply:
Yes...